lambdas/process-async-cri-credential/src/test/java/uk/gov/di/ipv/core/processasynccricredential/ProcessAsyncCriCredentialHandlerTest.java

```diff
@@ -385,22 +376,17 @@ private static void verifyBatchResponseFailures(
     private void mockCredentialIssuerConfig() {
         when(configService.getOauthCriActiveConnectionConfig(TEST_CREDENTIAL_ISSUER_ID))
                 .thenReturn(TEST_CREDENTIAL_ISSUER_CONFIG);
-        when(configService.getComponentId(ADDRESS.getId()))
-                .thenReturn(TEST_CREDENTIAL_ISSUER_CONFIG_ADDRESS.getComponentId());
-        when(configService.getComponentId(CLAIMED_IDENTITY.getId()))
-                .thenReturn(TEST_CREDENTIAL_ISSUER_CONFIG_CLAIMED_IDENTITY.getComponentId());
     }
 
-    private static OauthCriConfig createOauthCriConfig(String componentId)
-            throws URISyntaxException {
+    private static OauthCriConfig createTestOauthCriConfig() throws URISyntaxException {
         return OauthCriConfig.builder()
                 .tokenUrl(new URI(""))
                 .credentialUrl(new URI(""))
                 .authorizeUrl(new URI(""))
                 .clientId("ipv-core")
                 .signingKey(EC_PRIVATE_KEY_JWK)
                 .encryptionKey(null)
-                .componentId(componentId)
+                .componentId(ProcessAsyncCriCredentialHandlerTest.TEST_COMPONENT_ID)
```

Comment:
```suggestion
                .componentId(TEST_COMPONENT_ID)
```

Reply:
ah I think this must have been an automatic refator 🤦